scripts/prepare-release.sh

```diff
@@ -33,7 +33,11 @@ for arch in `ls -1 $BIN_DIR/`;do
 
     # Create a tar.gz of the binary
     echo "gzipping binary $source_file as $target_file"
-    tar -czvf $target_file.tar.gz --directory=$source_dir $source_filename
+    if ["$suffix" == ".exe"] then
```

Comment:
@rnapoles-rh There is one syntax error here and thats why the unit tests are failing https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_odo/4176/pull-ci-openshift-odo-master-unit/1333745962283175936#1:build-log.txt%3A433 . It should have `;` after checks like `if ["$suffix" == ".exe"]; then` . Could you please correct it.

Reply:
fixed, thank you.